### ISSUE TITLE
rmview with rm2fb installed makes the UI freeze

### ISSUE THREAD

**Issue by reactormonk** [bordaigorl/rmview#53]
As soon as I start rmview, it straight up makes the tablet go icicle, I have to reboot it. Nothing interesting via `journalctl -f` on the device. After uninstalling `rm2fb`, no more issues.

**Comment by bordaigorl** [comment]
As prominently (I hope) explained in the Readme:


> :warning: **For reMarkable 2 users** :warning::
> rMview should work out of the box with the stock firmware.
> If you use [`rm2fb`](https://github.com/ddvk/remarkable2-framebuffer) there are known compatibilities issues that are [being addressed](https://github.com/pl-semiotics/rM-vnc-server/issues/5).

**Comment by reactormonk** [comment]
RTFM I guess. Oops. Maybe add a small check that logs an error message? :sweat_smile:

**Comment by bordaigorl** [comment]
No worries!
I am actually waiting until the next release of the VNC server is released: that should solve this compatibility issue and also unblock some new features of rmview that are on hold since they depend on those changes to the server. This is imminent, so I thought adding warnings would not be necessary...
